h2/src/main/org/h2/command/Parser.java

```diff
@@ -7290,6 +7290,13 @@ private Prepared parseSet() {
         } else if (readIf("JAVA_OBJECT_SERIALIZER")) {
             readIfEqualOrTo();
             return parseSetJavaObjectSerializer();
+        } else if (readIf("IGNORE_CATALOGS")) {
+            readIfEqualOrTo();
+            // Simulate multiple catalog compatibility by just ignoring (IGNORE_CATALOGS=TRUE in the database URL)
+
+            database.setIgnoreCatalogs(readBooleanSetting());
+
+            return new NoOperation(session);
```

Comment:
You can't change this setting directly here. Parser shouldn't execute any commands.

Reply:
Ahh, therefore you create a command. ok

Reply:
Added to SetTypes, adapted Set